_cucumber/features/login_register/social_registration.feature

```diff
@@ -0,0 +1,58 @@
+Feature: Social registration
+
+  As a visitor on the developers.redhat.com website,
+  I want to register using my Github account,
+  So that I can use RHD services.
+
+  @rhd_social_test_user01 @keycloak_teardown @github_teardown @logout
+  Scenario: 1 - Registration using GitHub login which contains all mandatory information (first name, last name, unique email)
+    Given I am on the Registration page
+    When I register a new account using my GitHub account
+    Then I should be registered and logged in
+
+  @rhd_social_test_user01 @keycloak_social_teardown @github_teardown @logout
+  Scenario: 2 - Registration using GitHub login which contains all mandatory information (first name, last name, but email already exists). User links GitHub account to existing account
+    Given I am on the Registration page
+    When I try to link a GitHub account to an existing account
+    Then I should see a warning that the email is already registered
+     And I click on the Send Verification email button
+    Then I should receive an email containing a verify email link
+     And I navigate to the verify email link
+    Then I should be registered and logged in
+
+  @rhd_social_test_user02 @keycloak_teardown @github_teardown @logout
+  Scenario: 3 - Registration using GitHub login which contains all mandatory information (first name, last name, but email already exists). User changes email to create new account.
+    Given I am on the Registration page
+    When I try to link a GitHub account to an existing account
+    Then I should see a warning that the email is already registered
+     And I select to Choose another email
+     And I complete the required additional information with a new email address
+    Then I should be registered and logged in
+
+  @rhd_social_test_user01 @logout @github_teardown
+  Scenario: 4 - Registration using GitHub login which doesn't contain some mandatory information (first name, last name), email is unique. User is asked to fill in mandatory info during login.
```

Comment:
Is this scenario finished? I miss some steps there like filling in data and verifying user is registered and logged in.